File: integration_tests/test-packages/r/testpkguno/R/function_a.R

# Adding an internal function to be sure tests catch
# these suddenly getting picked up by analyze.R
.add_5 <- function(n){
    return(n + 5)
}

#' @name function_a
#' @title function_a
#' @description function_a
#' @export
function_a <- function(){
    return(rnorm(1))
}

File: integration_tests/test-packages/python/testpkguno/testpkguno/function_a.py
# Adding these imports to be sure that analyze.py
# isn't tricked into thinking they belong to this
# package
import math
from random import LOG4  # type: ignore


# Adding an internal function to be sure tests catch
# these suddenly getting picked up by analyze.py
def _add_5(n):
    return n + 5


def function_a():
    return math.pi + _add_5(LOG4)
